class-wc-facebookcommerce.php

```diff
@@ -128,6 +131,7 @@ public function init() {
 			add_filter( 'woocommerce_navigation_get_breadcrumbs', [ $this, 'wc_page_breadcrumbs_filter' ], 99 );
 
 			if ( \WC_Facebookcommerce_Utils::isWoocommerceIntegration() ) {
+				require_once __DIR__ . '/vendor/autoload.php';
```

Comment:
I see we have a related PR migrating to autoloader (#1919) - assuming that will get merged before this. Not sure if it matters, just clarifying that this PR doesn't introduce autoloader.

Reply:
I'm planning to merge #1919 and then remove the autoloader addition from this PR.